.github/workflows/e2e.yaml

```diff
@@ -51,4 +51,3 @@ jobs:
         run: |
           export DFX_WALLET_WASM=$GITHUB_WORKSPACE/target/wasm32-unknown-unknown/release/wallet-opt.wasm
           bats e2e/bash/*.bash
-
```

Comment:
We leave a trailing space at the end of files as a formatting convention. You can enable this in VSCode by searching in settings for `insert final newline`